tests/AbstractModelManagerTestCase.php

```diff
@@ -37,13 +38,13 @@ abstract class AbstractModelManagerTestCase extends TestCase
     protected $documentManager;
 
     /**
-     * @var ClassMetadataFactory&Stub
+     * @var ClassMetadataFactory&MockObject
      */
     protected $metadataFactory;
 
     protected function setUp(): void
     {
-        $this->metadataFactory = $this->createStub(ClassMetadataFactory::class);
+        $this->metadataFactory = $this->createMock(ClassMetadataFactory::class);
```

Comment:
Is `expects()` called on this mock?

Reply:
I assume it is, in TestCase which extends from this one.

Reply:
I don't know that file, but I don't want to block this PR.

Reply:
It was introduced in commit https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/commit/3d052206d29b1f699f41666495cfdf223d51b279

And there is some expectations: https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/commit/3d052206d29b1f699f41666495cfdf223d51b279#diff-1891e728773a2fd296e39c4326e06a331b13b4db0a2df95affc3405f13bc1a7bR180

Reply:
Thank you for pointing.

Reply:
By the way, looking at [`InvocationStubber`](https://github.com/sebastianbergmann/phpunit/blob/9553a43f61a85b377085da47be07f5f0f69a2555/src/Framework/MockObject/Builder/InvocationStubber.php) which is what [`Stub::method()` returns](https://github.com/sebastianbergmann/phpunit/blob/9553a43f61a85b377085da47be07f5f0f69a2555/src/Framework/MockObject/Stub.php#L15), if we call any `with*` method, it should be a mock.